### ISSUE TITLE
Content switcher: Document manual mode of selection

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon-website#3364]
Overall the content switcher is a well implemented and documented component. There are however a few things we identify in the comments of our PDF exports of the [Usage](https://ibm.box.com/s/t9lhd31mvvga5ks3xu49xb0opnha5vv4) and [Style](https://ibm.box.com/s/7ncuk4he7chgcehb0a3fm6jccqqikd1k) pages (which should be thoroughly reviewed for content updates). The most significant of these are:
- there is no mention of the content selection update variants: automatic and manual. They aren't mentioned in the Usage documentation nor illustrated on the Style page. (However, they are properly implemented on storybook.)
- There's not mention of the child content of the Content switcher (it's equivalent to a tabpanel).  That seems like a pretty large gap

**Comment by sstrubberg** [comment]
Whoever takes this on: Clarify last bullet with Gower.

**Comment by mbgower** [comment]
I think some design considerations and decisions are needed on that second bullet, @sstrubberg. It would be good to get @jeanservaas, @laurenmrice and others talking about it. Perhaps something to cover in either an Carbon Accessibility Guild session, or in one of the issue refinement calls @tay1orjones organizes?

**Comment by laurenmrice** [comment]
Closing in favor of tracking via https://github.com/carbon-design-system/carbon/issues/13521
